checker-workflow-wrapping-workflow.cwl

```diff
@@ -0,0 +1,134 @@
+cwlVersion: v1.0
+class: Workflow
+
```

Comment:
Doesn't have to be done immediately, but a small `doc:` section would be helpful 
i.e. this is a workflow that checks on the out put of bwa by running md5sum on the results

Example at
http://www.commonwl.org/user_guide/17-metadata/

Reply:
Thanks @denis-yuen, @baminou is off today, we will do that later.

Reply:
Added an issue for doc: #17